extras-requirements.txt

```diff
@@ -11,14 +11,13 @@
 boto3~=1.9, <1.17.107
 botocore>=1.20.106,<1.20.107
 aiobotocore~=1.4.0
-# aioitertools 0.9 and above requires typing-extensions>4, which conflicts with kfp
-aioitertools<0.9
+aioitertools
 s3fs~=2021.8.1
 # https://github.com/Azure/azure-sdk-for-python/issues/24765#issuecomment-1150310498
 msrest~=0.6.21
 # azure-core>=1.23.0 uses ParamSpec, and hence depends on typing_extensions>=4
-azure-core<1.23
-azure-storage-blob~=12.0
+azure-core>=1.23.0
+azure-storage-blob~=12.13.0
```

Comment:
Hm, are you limiting it to 12.13.x on purpose? If so, why?

Reply:
yea, was not sure about it myself. so you suggest to just install latest of azure-core/storage-blob?

Reply:
Hm, I think mlrun standard is `~=x.y` if there is no reason to do it differently.